torch/lib/THCS/generic/THCSTensor.c

```diff
@@ -297,4 +346,117 @@ void THCSTensor_(retain)(THCState *state, THCSTensor *self)
   THAtomicIncrementRef(&self->refcount);
 }
 
+void THCSTensor_(contiguous)(THCState *state, THCSTensor *self) {
+  if (self->contiguous) return;
+  THCSTensor_(reorder)(state, self);
+  self->contiguous = 1;
+}
+
+void THCSTensor_(markContiguous)(THCState *state, THCSTensor *self) {
```

Comment:
Is that method really needed? What's the use case? It seems quite unsafe for me.